packages/ember-simple-auth/tests/simple-auth/session-test.js

```diff
@@ -162,12 +162,12 @@ describe('Session', function() {
 
         it('persists its content in the store', function(done) {
           var _this = this;
-
+          this.store.persist({ secure: { authenticator: 'authenticator' }, someOther: 'property' });
```

Comment:
That doesn't work. 

    this.session.restore().then() 

Is not called in that case